### DISCUSSION TITLE
Requiring 2FA is a terrible policy.

### DISCUSSION THREAD

**Discussion by ghost** [community/community#64794] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Requiring 2FA is a terrible policy.
Not everyone uses a mobile device, and/or active service, or wants GitHub malware/adware/spyware on their personal device(phone).
Were I using this platform professionally a device and/or service would be supplied.
I do not use GitHub professionally, merely for hobbies, and contributions to projects outside of my own are almost always given outside of the platform.
The security of open source is having the source available to many people, bugs/bad code can be removed sooner, requiring 2FA doesn't prevent malicious code, or bugs, in any way, yet will only limit people from contributing when bad code is discovered.
2FA could have an Email option, could work through previously verified desktop app, mobile device should not be the only option to continue using the platform.
While security keys are novel, the only real world use I've seen of them are with sensitive information systems, not common to have laying around, and in most situations this type of use would not be approved.

**Comment by Petschko** [comment]
Yep, I will also leave. I don't care about this account enough for 2FA.

I just do some hobby projects, guess its time to find an other place or self host my codes
